packages/fxa-payments-server/Gruntfile.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+module.exports = function (grunt) {
+  grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
+    concat: {
+      ftl: {
+        src: [
+          'src/lib/l10n/branding.ftl',
+          'src/*.ftl',
+          'src/lib/*.ftl',
+          'src/routes/**/*.ftl',
+          'src/components/**/*.ftl',
```

Comment:
Nitpick: there may be a way to reference `'src/lib/l10n/branding.ftl'` first and then reference `src/**/*.ftl` while excluding the branding file to avoid this many paths. However this doesn't really matter since we'll eventually change this to `../fxa-shared/lib/l10n/branding.ftl', 'src/**/*.ftl'`.

Might also be good to leave a comment above `branding.ftl` (and not in the file since that'd get into the l10n repo) noting that the branding file is temporary until we can use the one from `fxa-shared`. Haven't filed an issue for that yet, I'll comment back here when I have (may be before or after this is merged).

Reply:
I agree, it's not as clean as I would like. I could do

```
concat: {
  ftl: {
    src: [
      // 'src/branding.ftl' is temporary and will be replaced
      //  with '../fxa-shared/lib/l10n/branding.ftl' in a later ticket
      //  This will require coordination with l10n to resolve
      // conflicting IDs for identical terms.
      'src/branding.ftl',
      'src/**/*.ftl',
    ],
    dest: 'public/locales/en/payments.ftl',
  },
```

I've tested with `yarn run merge-ftl` and this only includes one instance of branding.ftl (concatenated first) in the destination file.

Downside is that the strings shared by multiple components are included midway in the concatenated file rather than the top. Might not matter?

Reply:
Nice, that looks cleaner. I think we're going to end up going with `src/**/*.ftl` anyway once we swap out the branding file, so that seems great.

> Downside is that the strings shared by multiple components are included midway in the concatenated file rather than the top. Might not matter?

I think this is fine. It might "feel" more ideal to be first under "branding" but doesn't affect anything so long as the comments match up and make sense once concatenated.